.github/workflows/tests.yaml

```diff
@@ -1119,3 +1119,31 @@ jobs:
           password: ${{secrets.DOCKER_PASSWORD}}
       - name: Run chaos test
         run: ./read_repair.sh
+  update-stability:
+    name: Recall Stability on object updates
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    env:
+      PERSISTENCE_LSM_ACCESS_STRATEGY: ${{inputs.lsm_access_strategy}}
+      PROMETHEUS_MONITORING_ENABLED: "true"
+      DATASET: "dbpedia-100k-openai-ada002-angular"
+      DISTANCE: "cosine"
+      INDEX_TYPE: "hnsw"
+      UPDATE_PERCENTAGE: "0.08"
+      CLEANUP_INTERVAL_SECONDS: "30"
+      UPDATE_ITERATIONS: "10"
+      REQUIRED_RECALL: "0.992"
+    steps:
+      - uses: actions/checkout@v3
+      # - name: Polar Signals Continuous Profiling
+      #   uses: polarsignals/gh-actions-ps-profiling@v0.0.1
+      #   with:
+      #     polarsignals_cloud_token: ${{ secrets.POLARSIGNALS_TOKEN }}
+      #     labels: 'job=${{ github.job }};gh_run_id=${{ github.run_id }}'
```

Comment:
**suggestion (non-blocking):** do we need this part ?

Reply:
In most cases we would not need it, but it seems most of the jobs do have it commented out just in case something goes wrong and we need to see a continuous profile to debug it.